src/cssname.h

```diff
@@ -50,7 +50,6 @@ class CssName {
         index_ = static_cast<size_t>(builtin);
     }
     CssName(const std::string& name);
-    CssName(const CssName& other) = default;
```

Comment:
What happened here?

Reply:
I saw it in one of the changes made in your PR and then one of the compiler errors suggested to remove the default constructor here (and fortunately, we do not seem to need it).

Reply:
Oh I see, then let's kill it :)